Public/index.html

```diff
@@ -13,22 +13,25 @@
     <div class="card">
       <div class="container_copy">
     <h1>
-      <img src="images/dooooL.png" alt="To-Do Icon" class="todo-icon">
+      <img src="images/dooooL.png" alt="To-Do Icon" class="todo-icon " >
       
       To-Do List!
      
       
     </h1>
     
     <div class="row">
-    <input type="text" id="task-input" placeholder="Add a new task" />
+    <input type="text" id="task-input" placeholder="Add a new task" maxlength="20"/>

```

Comment:
@MbathaSimphiwe I added the character limit for the tasks and set it up to accept 20 characters.
![cha limit](https://github.com/user-attachments/assets/a913fd78-9530-43c7-9dd3-6b4a4b4156b4)

Reply:
@Valerietshuma, Awesome, now its looks vey good.